java/cdk/src/main/java/sleeper/cdk/stack/CompactionStack.java

```diff
@@ -276,7 +278,7 @@ private void lambdaToCreateCompactionJobsBatchedViaSQS(
                 .timeout(Duration.seconds(instanceProperties.getInt(TABLE_BATCHING_LAMBDAS_TIMEOUT_IN_SECONDS)))
                 .handler("sleeper.compaction.job.creation.lambda.CreateCompactionJobsTriggerLambda::handleRequest")
                 .environment(environmentVariables)
-                .reservedConcurrentExecutions(1)
+                .reservedConcurrentExecutions(instanceProperties.getInt(COMPACTION_LAMBDA_CONCURRENCY_RESERVED))
```

Comment:
This should be set on the handler function. The trigger function needs this set to 1.

The trigger function's job is just to send messages on the SQS FIFO queue that triggers the handler function. It runs on a cron-like schedule and sends one message for each table that needs processing. This is so that the handler function can process the tables in parallel.

Reply:
Addressed